pkg/argparse/args.go

```diff
@@ -0,0 +1,117 @@
+package argparse
+
+import (
+	"flag"
+	"os"
+	"strings"
+
+	"github.com/kitex-contrib/kitexcall/pkg/config"
+	"github.com/kitex-contrib/kitexcall/pkg/errors"
+	"github.com/kitex-contrib/kitexcall/pkg/log"
+)
+
+type Augment struct {
+	config.Config
+}
+
+type EndpointList []string
+
+func (e *EndpointList) String() string {
+	return strings.Join(*e, ",")
+}
+
+func (e *EndpointList) Set(value string) error {
+	*e = append(*e, value)
+	return nil
+}
+
+func (a *Augment) buildFlags() *flag.FlagSet {
+	f := flag.NewFlagSet(os.Args[0], flag.ContinueOnError)
+	f.StringVar(&a.Type, "type", "unknown", "Specify the type of IDL: 'thrift' or 'protobuf'.")
+	f.StringVar(&a.Type, "t", "unknown", "Specify the type of IDL: 'thrift' or 'protobuf'. (shorthand)")
+
+	f.StringVar(&a.IDLPath, "idl-path", "", "Specify the path of IDL file.")
+	f.StringVar(&a.IDLPath, "p", "", "Specify the path of IDL file. (shorthand)")
+
+	f.StringVar(&a.Service, "service", "", "Specify the service name.")
+	f.StringVar(&a.Service, "s", "", "Specify the service name. (shorthand)")
+
+	f.StringVar(&a.Method, "method", "", "Specify the method name.")
+	f.StringVar(&a.Method, "m", "", "Specify the method name. (shorthand)")
+
+	f.StringVar(&a.Data, "data", "", "Specify the data to be sent.")
+	f.StringVar(&a.Data, "d", "", "Specify the data to be sent.")
+
+	f.Var((*EndpointList)(&a.Endpoint), "endpoint", "Specify the server endpoints. Can be repeated.")
+	f.Var((*EndpointList)(&a.Endpoint), "e", "Specify the server endpoints. Can be repeated. (shorthand)")
+
+	f.BoolVar(&a.Verbose, "verbose", false, "Enable verbose mode.")
+	f.BoolVar(&a.Verbose, "v", false, "Enable verbose mode. (shorthand)")
+
+	f.StringVar(&a.Transport, "transport", "", "Specify the transport type. Can be 'TTHeader', 'Framed', 'TTHeaderFramed'.")
+
+	return f
+}
+
+func (a *Augment) ParseArgs() error {
+	f := a.buildFlags()
+	if err := f.Parse(os.Args[1:]); err != nil {
+		return errors.New(errors.ArgParseError, "Flag parse error: %v", err)
+	}
+	log.Verbose = a.Verbose
+	log.Info("Parse args succeed")
```

Comment:
What's the default log level?

Reply:
If the user does not specify the -v option, then no info will be output